idunn/blocks/opening_hour.py

```diff
@@ -0,0 +1,77 @@
+import re
+import logging
+import datetime
+from apistar import types, validators
+import humanized_opening_hours as hoh
+from humanized_opening_hours.exceptions import (ParseError, SpanOverMidnight)
+
+def from_hours(hbnt):
+    """
+    Takes the "hours before next change" string as input and returns an int of the number of seconds.
+
+    >>> from_hours("")
+    0
+    >>> from_hours("5 minutes")
+    300
+    >>> from_hours("5 hours and 35 minutes")
+    20100
+    """
+    seconds = 0
+
+    hours = re.search(r"(\w+) hours", hbnt)
+    if hours:
+        seconds += int(hours.group(1)) * 3600
+
+    minutes = re.search(r"(\w+) minutes", hbnt)
+    if minutes:
+        seconds += int(minutes.group(1)) * 60
+    return seconds
+
+class OpeningHourBlock(types.Type):
+    type = validators.String(title="type")
+    status = validators.String()
+    next_transition_time = validators.String()
+    seconds_before_next_transition = validators.Integer()
+    is_24_7 = validators.Boolean()
+    raw = validators.String()
+    days = validators.Array()
+
+    @classmethod
+    def from_es(cls, es_poi, lang):
+        logger = logging.getLogger('idunn')
+        is247 = False
+
+        raw = es_poi.get('properties', {}).get('opening_hours')
+        if raw is None:
+            return None
+        elif raw is '24/7':
```

Comment:
you can rewrite this: 
```python
if raw is None:
    return None
is247  = raw == '24/7'
```
(for this use it seems more appropriate to use `==` than `is`)

Reply:
done